package.json

```diff
@@ -14,13 +14,10 @@
     "node": ">=20.18.3"
   },
   "scripts": {
-    "install-all": "npm install --workspaces",
-    "build": "npm run build --workspaces",
-    "test": "npm run test --workspaces",
-    "clean": "npm run clean --workspaces",
-    "fix": "stylelint packages/test-repository/force-app/main/default/lwc/myComponent/test.css --fix",
-    "report": "node node_modules/@salesforce-ux/stylelint-plugin-slds/build/report.js -d packages/test-repository",
-    "publish-all": "node scripts/release.js"
+    "preinstall": "node -e \"if(process.env.npm_execpath.indexOf('yarn') === -1) throw new Error('You must use Yarn to install dependencies')\"",
```

Comment:
Do you think we need this kind of check for node version also? Like when we run thie `slds-linter`, we check the current node version of the user and throw and error if it is not meeting our expectations?